Build  failed when i not strart up docker

```
[INFO] Running org.apache.shardingsphere.integration.data.pipline.cases.mysql.MySQLManualScalingTest
[INFO ] 2022-04-25 13:01:59.646 [main] o.t.d.DockerClientProviderStrategy - Loaded org.testcontainers.dockerclient.UnixSocketClientProviderStrategy from ~/.testcontainers.properties, will try it first
[INFO ] 2022-04-25 13:02:30.084 [main] o.t.d.DockerMachineClientProviderStrategy - docker-machine executable was not found on PATH ([/usr/local/bin, /usr/local/sbin, /usr/local/bin, /usr/bin, /bin, /usr/sbin, /sbin, /Users/zhaojinchao/Documents/apache-maven-3.8.1/bin])
[ERROR] 2022-04-25 13:03:00.089 [main] o.t.d.DockerClientProviderStrategy - Could not find a valid Docker environment. Please check configuration. Attempted configurations were:
[ERROR] 2022-04-25 13:03:00.089 [main] o.t.d.DockerClientProviderStrategy -     UnixSocketClientProviderStrategy: failed with exception TimeoutException (Timeout waiting for result with exception). Root cause IOException (native connect() failed : Connection refused)
[ERROR] 2022-04-25 13:03:00.089 [main] o.t.d.DockerClientProviderStrategy -     UnixSocketClientProviderStrategy: failed with exception TimeoutException (Timeout waiting for result with exception). Root cause IOException (native connect() failed : Connection refused)
[ERROR] 2022-04-25 13:03:00.089 [main] o.t.d.DockerClientProviderStrategy - As no valid configuration was found, execution cannot continue
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 60.744 s <<< FAILURE! - in org.apache.shardingsphere.integration.data.pipline.cases.mysql.MySQLManualScalingTest
[ERROR] assertManualScalingSuccess(org.apache.shardingsphere.integration.data.pipline.cases.mysql.MySQLManualScalingTest)  Time elapsed: 0.007 s  <<< ERROR!
java.lang.IllegalStateException: Could not find a valid Docker environment. Please see logs and check configuration
        at org.testcontainers.dockerclient.DockerClientProviderStrategy.lambda$getFirstValidStrategy$7(DockerClientProviderStrategy.java:215)
        at java.util.Optional.orElseThrow(Optional.java:290)
        at org.testcontainers.dockerclient.DockerClientProviderStrategy.getFirstValidStrategy(DockerClientProviderStrategy.java:207)
        at org.testcontainers.DockerClientFactory.getOrInitializeStrategy(DockerClientFactory.java:136)
        at org.testcontainers.DockerClientFactory.client(DockerClientFactory.java:178)
        at org.testcontainers.LazyDockerClient.getDockerClient(LazyDockerClient.java:14)
        at org.testcontainers.LazyDockerClient.authConfig(LazyDockerClient.java:12)
        at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:310)
        at org.apache.shardingsphere.test.integration.framework.container.atomic.DockerITContainer.start(DockerITContainer.java:49)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
        at java.util.LinkedList$LLSpliterator.forEachRemaining(LinkedList.java:1235)
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
        at org.apache.shardingsphere.test.integration.framework.container.atomic.ITContainers.start(ITContainers.java:75)
        at org.apache.shardingsphere.integration.data.pipline.container.compose.BaseComposedContainer.start(BaseComposedContainer.java:59)
        at org.apache.shardingsphere.integration.data.pipline.container.compose.LocalComposedContainer.start(LocalComposedContainer.java:43)
        at org.apache.shardingsphere.integration.data.pipline.cases.BaseScalingITCase.<init>(BaseScalingITCase.java:71)
        at org.apache.shardingsphere.integration.data.pipline.cases.mysql.BaseMySQLScalingCase.<init>(BaseMySQLScalingCase.java:36)
        at org.apache.shardingsphere.integration.data.pipline.cases.mysql.MySQLManualScalingTest.<init>(MySQLManualScalingTest.java:39)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)
        at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
        at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
        at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
        at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
        at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Errors: 
[ERROR]   MySQLManualScalingTest.<init>:39->BaseMySQLScalingCase.<init>:36->BaseScalingITCase.<init>:71 » IllegalState
[INFO] 
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
[INFO] 

```

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/BaseScalingITCase.java
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/BaseScalingITCase.java
@@ -18,7 +18,6 @@
 package org.apache.shardingsphere.integration.data.pipline.cases;
 
 import com.google.common.base.Joiner;
-import com.google.common.base.Splitter;
 import com.google.common.collect.Sets;
 import lombok.AccessLevel;
 import lombok.Getter;
@@ -87,7 +86,7 @@ public abstract class BaseScalingITCase {
                 connection.createStatement().execute(String.format(commonSQLCommand.getAddResource(), dbIndex, getDatabaseUrl(), value));
                 dbIndex++;
             }
-            for (String sql : Splitter.on(";").splitToList(commonSQLCommand.getCreateShardingAlgorithm()).stream().filter(StringUtils::isNotBlank).collect(Collectors.toList())) {
+            for (String sql : commonSQLCommand.getCreateShardingAlgorithm()) {
                 connection.createStatement().execute(sql);
             }
             // TODO sleep to wait for sharding algorithm table created，otherwise, the next sql will fail.

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/command/CommonSQLCommand.java
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/command/CommonSQLCommand.java
@@ -23,6 +23,7 @@ import javax.xml.bind.annotation.XmlAccessType;
 import javax.xml.bind.annotation.XmlAccessorType;
 import javax.xml.bind.annotation.XmlElement;
 import javax.xml.bind.annotation.XmlRootElement;
+import java.util.List;
 
 @Data
 @XmlRootElement(name = "command")
@@ -39,7 +40,7 @@ public final class CommonSQLCommand {
     private String useDatabase;
     
     @XmlElement(name = "create-sharding-algorithm")
-    private String createShardingAlgorithm;
+    private List<String> createShardingAlgorithm;
     
     @XmlElement(name = "create-sharding-table")
     private String createShardingTable;

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/command/CreateTableSQLCommand.java
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/command/CreateTableSQLCommand.java
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.shardingsphere.integration.data.pipline.cases.command.mysql;
+package org.apache.shardingsphere.integration.data.pipline.cases.command;
 
 import lombok.Data;
 
@@ -27,7 +27,7 @@ import javax.xml.bind.annotation.XmlRootElement;
 @Data
 @XmlRootElement(name = "command")
 @XmlAccessorType(XmlAccessType.FIELD)
-public final class MySQLCommand {
+public final class CreateTableSQLCommand {
     
     @XmlElement(name = "create-table-order")
     private String createTableOrder;

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/mysql/BaseMySQLScalingCase.java
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/mysql/BaseMySQLScalingCase.java
@@ -21,7 +21,7 @@ import lombok.Getter;
 import org.apache.commons.lang3.tuple.Pair;
 import org.apache.shardingsphere.infra.database.type.dialect.MySQLDatabaseType;
 import org.apache.shardingsphere.integration.data.pipline.cases.BaseScalingITCase;
-import org.apache.shardingsphere.integration.data.pipline.cases.command.mysql.MySQLCommand;
+import org.apache.shardingsphere.integration.data.pipline.cases.command.CreateTableSQLCommand;
 import org.apache.shardingsphere.integration.data.pipline.util.TableCrudUtil;
 
 import javax.xml.bind.JAXB;
@@ -30,16 +30,16 @@ import java.util.List;
 public abstract class BaseMySQLScalingCase extends BaseScalingITCase {
     
     @Getter
-    private final MySQLCommand mySQLCommand;
+    private final CreateTableSQLCommand createTableSQLCommand;
     
     public BaseMySQLScalingCase() {
         super(new MySQLDatabaseType());
-        mySQLCommand = JAXB.unmarshal(BaseMySQLScalingCase.class.getClassLoader().getResource("env/mysql/sql.xml"), MySQLCommand.class);
+        createTableSQLCommand = JAXB.unmarshal(BaseMySQLScalingCase.class.getClassLoader().getResource("env/mysql/sql.xml"), CreateTableSQLCommand.class);
     }
     
     protected void initTableAndData() {
-        getJdbcTemplate().execute(mySQLCommand.getCreateTableOrder());
-        getJdbcTemplate().execute(mySQLCommand.getCreateTableOrderItem());
+        getJdbcTemplate().execute(createTableSQLCommand.getCreateTableOrder());
+        getJdbcTemplate().execute(createTableSQLCommand.getCreateTableOrderItem());
         Pair<List<Object[]>, List<Object[]>> dataPair = TableCrudUtil.generateInsertDataList(3000);
         getJdbcTemplate().batchUpdate(getCommonSQLCommand().getInsertOrder(), dataPair.getLeft());
         getJdbcTemplate().batchUpdate(getCommonSQLCommand().getInsertOrderItem(), dataPair.getRight());

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/mysql/MySQLManualScalingCase.java
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/cases/mysql/MySQLManualScalingCase.java
@@ -36,7 +36,7 @@ import static org.junit.Assert.assertThat;
  * MySQL manual scaling test case.
  */
 @Slf4j
-public final class MySQLManualScalingTest extends BaseMySQLScalingCase {
+public final class MySQLManualScalingCase extends BaseMySQLScalingCase {
     
     private Thread increaseTaskThread;
     

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/container/compose/DockerComposedContainer.java
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipline/container/compose/DockerComposedContainer.java
@@ -53,8 +53,8 @@ public final class DockerComposedContainer extends BaseComposedContainer {
     public DataSource getProxyDataSource(final String databaseName) {
         HikariDataSource result = new HikariDataSource();
         result.setDriverClassName(DataSourceEnvironment.getDriverClassName(getDatabaseContainer().getDatabaseType()));
-        String jdbcUrl = StringUtils.appendIfMissing(DataSourceEnvironment.getURL(getDatabaseContainer().getDatabaseType(), getProxyContainer().getHost(), getProxyContainer().getFirstMappedPort(), databaseName), "&rewriteBatchedStatements=true");
-        result.setJdbcUrl(jdbcUrl);
+        String jdbcUrl = DataSourceEnvironment.getURL(getDatabaseContainer().getDatabaseType(), getProxyContainer().getHost(), getProxyContainer().getFirstMappedPort(), databaseName);
+        result.setJdbcUrl(StringUtils.appendIfMissing(jdbcUrl, "&rewriteBatchedStatements=true"));
         result.setUsername("root");
         result.setPassword("root");
         result.setMaximumPoolSize(2);

--- a/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/common/command.xml
+++ b/shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/common/command.xml
@@ -29,12 +29,18 @@
     </add-resource>
     
     <create-sharding-algorithm>
-        CREATE SHARDING ALGORITHM database_inline (
-        TYPE(NAME=INLINE,PROPERTIES("algorithm-expression"="ds_${user_id % 2}")));
-        CREATE SHARDING ALGORITHM t_order_inline (
-        TYPE(NAME=INLINE,PROPERTIES("algorithm-expression"="t_order_${order_id % 2}")));
         CREATE SHARDING ALGORITHM t_order_item_inline (
         TYPE(NAME=INLINE,PROPERTIES("algorithm-expression"="t_order_item_${order_id % 2}")))
+    </create-sharding-algorithm>
+    
+    <create-sharding-algorithm>
+        CREATE SHARDING ALGORITHM database_inline (
+        TYPE(NAME=INLINE,PROPERTIES("algorithm-expression"="ds_${user_id % 2}")))
+    </create-sharding-algorithm>
+    
+    <create-sharding-algorithm>
+        CREATE SHARDING ALGORITHM t_order_inline (
+        TYPE(NAME=INLINE,PROPERTIES("algorithm-expression"="t_order_${order_id % 2}")))
     </create-sharding-algorithm>
     
     <create-sharding-table>

